pylint/config/option_manager_mixin.py

```diff
@@ -274,7 +274,10 @@ def read_config_file(self, config_file=None, verbose=None):
             self.set_current_module(config_file)
             parser = self.cfgfile_parser
             if config_file.endswith(".toml"):
-                self._parse_toml(config_file, parser)
+                try:
+                    self._parse_toml(config_file, parser)
+                except toml.TomlDecodeError as e:
+                    self.add_message("conf-parse-error", line=0, args=str(e))
```

Comment:
```suggestion
                    self.add_message("config-parse-error", line=0, args=str(e))
```

Might as well add those two additional letters :)